### PR TITLE
Problem: ethermint develop branch is not used

### PR DESCRIPTION
**Pull request by mmsqe**
👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -466,15 +466,18 @@ func New(
...
 	// Create Ethermint keepers
+	feeMarketSs := app.GetSubspace(feemarkettypes.ModuleName)
 	app.FeeMarketKeeper = feemarketkeeper.NewKeeper(
-		appCodec, app.GetSubspace(feemarkettypes.ModuleName), keys[feemarkettypes.StoreKey], tkeys[feemarkettypes.TransientKey],
+		appCodec, authtypes.NewModuleAddress(govtypes.ModuleName),
+		keys[feemarkettypes.StoreKey], tkeys[feemarkettypes.TransientKey], feeMarketSs,
 	)
-
+	// Set authority to x/gov module account to only expect the module account to update params
+	evmSs := app.GetSubspace(evmtypes.ModuleName)
```
Comment (JayT106) [inline app/app.go:475]: same here, maybe uses SS instead of Ss or just S

File: x/cronos/keeper/grpc_query.go [hunk 2]
```diff
@@ -5,12 +5,14 @@ import (
 	"fmt"
 	"math/big"
 
+	errorsmod "cosmossdk.io/errors"
```
Comment (JayT106) [inline x/cronos/keeper/grpc_query.go:8]: maybe uses `sdkerr` ?

**Review by JayT106 — APPROVED** [review]
LGTM, 
the ibc go has released v6.2. maybe uses this instead of v6.1 ?

File: x/cronos/keeper/grpc_query.go (hunk 2 above)
Comment (mmsqe) [reply x/cronos/keeper/grpc_query.go:8]: we might gradually move to errorsmod instead of sdkerrors

File: go.mod [hunk 3]
```diff
@@ -229,8 +230,7 @@ replace (
 	// dgrijalva/jwt-go is deprecated and doesn't receive security updates.
 	// TODO: remove it: https://github.com/cosmos/cosmos-sdk/issues/13134
 	github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt/v4 v4.4.2
-	github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethereum v1.10.19-deepcopy-jumptable
-	github.com/evmos/ethermint => github.com/crypto-org-chain/ethermint v0.0.0-20230605085654-2136ad029860
+	github.com/evmos/ethermint => github.com/crypto-org-chain/ethermint v0.0.0-20230615084816-d14eadaa30e6
```
Comment (yihuang) [inline go.mod:233]: in the long run, we should simply change the package path ;D

Comment (mmsqe) [reply go.mod:233]: not sure to which package?

Comment (yihuang) [reply go.mod:233]: I mean import `github.com/crypto-org-chain/ethermint` directly, instead of adding replacements.

**Review by yihuang — APPROVED** [review]

File: app/app.go (hunk 1 above)
Comment (yihuang) [reply app/app.go]: `S` is for `Subspace`? I think it follows the naming from ethermint repo

**Pull request merged**